File: NataliaCopete_GenerarTiempos.py
import numpy as np
import matplotlib.pyplot as plt



def fibonacci(N):
  if (N==0 or N==1):
    return N
  else:
    return fibonacci(N-2)+fibonacci(N-1)


def get_time(N):
   import time
   t0 = time.time()
   fibonacci(N)
   t1 = time.time()-t0
   return t1

for i in range(0,40):

  print(i,get_time(i))

File: NataliaCopete_GenerarTiempos.cpp
#include <iostream>
#include<ctime> 

using std::cout;
using std::endl;



int fibonacci(int N);
float get_time(int N);

int main(){
 int i;
 for(i=0;i<40;i++){

cout<<i<<" "<<get_time(i)<<endl;
}
return 0;
}


int fibonacci(int N){
  if(N==0||N==1){return N;
  } 
  else{ return fibonacci(N-2)+fibonacci(N-1);
  }
}

float get_time(int N){
clock_t t;
t = clock();
fibonacci(N);
t = clock() - t;
float time = ((float)t)/CLOCKS_PER_SEC;
return time;
}
